smsgateway/backends/jasmin.py

```diff
@@ -42,11 +48,11 @@ def send(self, sms_request, account_dict):
             url = self.get_send_url(request, account_dict)
 
             # Make request to provider
-            result = u''
+            result = ''
             if url is not None:
                 try:
                     sock = urlopen(url)
-                    result = sock.read()
+                    result = sock.read().decode('utf_8')
```

Comment:
Side note: `utf-8` should be default, so not really required here. Given that we're playing with encodings, it makes sense though.